integration-tests/artifacts/gatsby-ssr.js

```diff
@@ -10,3 +10,23 @@ export function onRenderBody({ setHeadComponents }) {
     />
   )
 }
+
+export function wrapPageElement({ element }) {
+  return (
+    <>
+      <h1>(to match gatsby-browser structure without using same imports)</h1>
+      {element}
+    </>
+  )
+}
+
+export function wrapRootElement({ element }) {
+  return (
+    <>
+      <div>
+        (to match src/components/github.js structure without using same imports)
+      </div>
+      {element}
+    </>
+  )
+}
```

Comment:
this is just to make sure hydration works (even tho text itself doesn't match, html elements structure does). it doesn't matter for tests we do (as we don't do something like cypress in here), but for manual checking it's useful for hydration to actually render things properly